platform/security/pdp/security-pdp-authzrealm/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -15,10 +15,16 @@
 <blueprint xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
            xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
 
-    <bean id="pdpAuthzRealm" class="ddf.security.pdp.realm.SimpleAuthzRealm">
-        <cm:managed-properties persistent-id="ddf.security.pdp.realm.SimpleAuthzRealm"
+    <bean id="pdpAuthzRealm" class="ddf.security.pdp.realm.AuthzRealm">
+        <cm:managed-properties persistent-id="ddf.security.pdp.realm.AuthzRealm"
                                update-strategy="container-managed"/>
+        <argument value="etc/pdp/policies"/>
         <property name="expansionServiceList" ref="userAttributeExpansionList"/>
+        <property name="matchOneMappings">
+            <list>
+                <value>http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=invalid-state</value>
```

Comment:
I agree. I can write up a ticket to move this to a PolicyExtension. Now that we've merged the PDPs together, that will actually work as intended.